docs/developer/payments/transactions.mdx

```diff
@@ -371,6 +371,61 @@ sequenceDiagram
 
 Follow [Transactions Webhook Events](/developer/extending/webhooks/synchronous-events/transaction.mdx) guide.
 
+### Automatic checkout completion
+
+The checkout process can be automatically completed based on the transaction events, once the full payment is made.
+
+A checkout is considered fully paid when the [`checkout.charge_status`](/api-reference/checkout/objects/checkout#checkoutchargestatuscheckoutchargestatusenum---) 
```

Comment:
If this is the case, then I think our logic is incorrect: if channel uses `transactionFlowStrategy.AUTHORIZATION`, then "paid" = `authorizeStatus.FULL`